aep/general/0134/aep.md.j2

```diff
@@ -57,7 +59,28 @@ separator, `snake_case` is used.
 
 {% tab oas %}
 
-**Note:** OAS example not yet written.
+```yaml
+paths:
+  /v1/publishers/{publisher}/books:
+    patch:
+      summary: Update a book
+      operationId: publishers.book:create
```

Comment:
There is no guidance for OAS operationId, only general "method name" guidance (which should probably be moved into the proto tab, but that is a different discussion). However, in [AEP-131](https://aep.dev/131) there is such guidance and extrapolating from it, I would think this should be 

```suggestion
      operationId: updateBook
```
It definitely shouldn't be anything with create in it.

Reply:
:thinking: well, I don't know if we've actually made a decision about the operationId yet, and I think that it's usage will be very critical when we start building clients.

For now I'll remove it, and will probably do a follow-up to determine the standard around OAS operations IDs.

Reply:
here's a tracking issue: https://github.com/aep-dev/aep.dev/issues/208

Reply:
apologies for the delay getting to these comments - agreed there is no explicit guidance here, and I've been removing the setting of `operationId` from the examples since there isn't a standard that AEP suggests yet.

If you have a proposal, feel free to add it to issue above!